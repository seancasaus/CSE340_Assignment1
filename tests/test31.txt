M5h3 ((((_)*).((b)*))*).((q)|(p)) , QR4tu ((_).(b)).((Q).(((_)|(r))|((p)*)))  , JK4oiu ((_).((_)*)).((m)|(n)) #
" bbbpqbpppmnbQrmpq "
