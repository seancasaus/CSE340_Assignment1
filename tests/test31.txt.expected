M5h3 , "bbbp"
M5h3 , "q"
M5h3 , "bp"
M5h3 , "p"
M5h3 , "p"
JK4oiu , "m"
JK4oiu , "n"
QR4tu , "bQr"
JK4oiu , "m"
M5h3 , "p"
M5h3 , "q"
